benchexec/tools/coveritest.py

```diff
@@ -0,0 +1,25 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.tools.cpachecker as cpachecker
+
+
+class Tool(cpachecker.Tool):
+    """Tool info for CoVeriTest."""
+
+    REQUIRED_PATHS = [
```

Comment:
This duplicates the info from CPAchecker's tool-info module. I would suggest to deduplicate this, otherwise CoVeriTest's tool-info module is likely to become outdated when something in CPAchecker requires adding a new item to this list.